### PR TITLE
Change Spaceport descriptions in response to Remembrance Day

### PR DESCRIPTION
**Pull request by Anarchist2**
## Summary
This PR adds a new set of events that change spaceport descriptions in the Dirt Belt during the lead up to Remembrance Day, before reverting them to normal. Also modified are the spaceport descriptions of Allhome and Valhalla, while Tundra is left unaltered due to being affected by other events. New Washington also has a permanent change to its spaceport description.

### CODE REVIEW

File: data/human/events.txt [hunk 1]
```diff
@@ -31,6 +31,146 @@ mission "remembrance day"
...
+	planet "Bounty"
+		add spaceport `	The local residents are also setting up decorations in the marketplace in preparation for Remembrance Day.`
+	planet "Clark"
+		add spaceport `	This heightened mood carries over into the planet's Remembrance Day festivities. All around the spaceport, you can hear songs echo and blend.`
+	planet "Gemstone"
+		add spaceport `	In commemoration of Remembrance Day, the local residents have set up a few flags of in the spaceport. You doubt that anybody here could afford any more than what is already set up.`
+	planet "Heartland"
+		add spaceport `	One section of the tractor showcase appears to currently be dedicated to antique tractors back from the Alpha Wars. On one landing pad, you spot some dock workers unloading a gigantic tractor out from a bulk freighter's hold.`
+	planet "Hopper"
+		add spaceport `	The inn has been repurposed for hosting Remembrance Day celebrations. A canvas just above its entrance reads, "For those who have fallen, and those who will fall." The canvas is already dappled.`
```
Comment (EjoThims) [inline data/human/events.txt:68]: "The canvas is already dappled."

The context of this is unclear to me.

File: data/human/events.txt [hunk 2]
```diff
@@ -31,6 +31,146 @@ mission "remembrance day"
...
+	planet "Ingot"
+		add spaceport `	The decorations for Remembrance Day on this planet amount to a single batch of flags placed in the middle of the trading hub.`
+	planet "New Argentina"
+		add spaceport `	The Grange is currently set up for Remembrance Day celebrations. The population inside consists mostly of the older residents of the planet, many of them likely having traveled tens of thousands of kilometers to arrive here.`
+	planet "New Austria"
+		add spaceport `	The town is abuzz in preparation for Remembrance Day. Decorations have been set up on every building you can see, and you can hear songs coming from the town's tavern.`
+	planet "New Boston"
+		add spaceport `	A smaller tent has been set up beside the market. Inside are various pamphlets for Remembrance Day events occuring on New Boston.`
+	planet "New Britain"
+		add spaceport `	In celebration of the upcoming Remembrance Day, the night sky above the port is densely occupied by choreographed drone formations, starship stunt flying and firework showcases.`
```
Comment (EjoThims) [inline data/human/events.txt:78]:
```suggestion
		add spaceport `	In celebration of the upcoming Remembrance Day, the night sky above the port is densely occupied by choreographed drone formations, starship stunt flying, and firework showcases.`
```

Oxford comma is a hill I will die on. :P

File: data/human/events.txt [hunk 3]
```diff
@@ -31,6 +31,146 @@ mission "remembrance day"
...
+	planet "New Greenland"
+		add spaceport `	New Greenland was one of the worlds that was occupied during the Alpha Wars. Despite this, the celebrations on New Greenland are quite humble, most of the locals simply going about business as usual.`
```
Comment (EjoThims) [inline data/human/events.txt:80]:
```suggestion
		add spaceport `	New Greenland was one of the worlds that was occupied during the Alpha Wars. Despite this, the Remembrance Day celebrations on New Greenland are quite humble, most of the locals simply going about business as usual.`
```

File: data/human/events.txt [hunk 4]
```diff
@@ -31,6 +31,146 @@ mission "remembrance day"
...
+	planet "New Holland"
+		add spaceport `	Modest Remembrance Day decorations have been set up around the spaceport, and in the distance, you can see marginally more lavish decorations set up within the main city.`
+	planet "New Iceland"
+		add spaceport `	The caustic climate makes it difficult for the local populace to celebrate Remembrance Day properly, though a small entourage has gathered in the cafeteria regardless.`
+	planet "New India"
+		add spaceport `	Despite the relative recency of settlement, Remembrance Day celebrations still appear to be occuring, with Alpha War shanties audible from one of the spaceport buildings.`
+	planet "New Kansas"
+		add spaceport `	A group of locals are gathered in a tavern in the western part of the spaceport. The building is decorated with Remembrance Day memorabilia, and you can see several people showing off knitted patterns inside.`
+	planet "New Portland"
+		add spaceport `	It appears that the spaceport has not been embellished for the upcoming Remembrance Day. You assume that the main preparations are taking place in the planet's towns proper.`
```
Comment (EjoThims) [inline data/human/events.txt:90]:
```suggestion
		add spaceport `	It appears that the spaceport has not been embellished for the upcoming Remembrance Day. You assume that the main preparations are taking place in the planet's individual towns.`
```

File: data/human/events.txt [hunk 5]
```diff
@@ -31,6 +31,146 @@ mission "remembrance day"
...
+	planet "New Sahara"
+		add spaceport `	The living quarters have been modestly decorated for Remembrance Day, with ceremonial flags being set up inside the dining halls, and a showcase of medals being set up on a wall, each one belonging to the ancestors of those who live here now.`
```
Comment (EjoThims) [inline data/human/events.txt:92]:
```suggestion
		add spaceport `	The living quarters have been modestly decorated for Remembrance Day, with ceremonial flags being set up inside the dining halls and a showcase of medals being set up on a wall, each one belonging to the ancestors of those who live here now.`
```

File: data/human/events.txt [hunk 6]
```diff
@@ -31,6 +31,146 @@ mission "remembrance day"
...
+		add spaceport `	Compared to the rest of the Dirt Belt, the people of Oblivion are not particularily keen on celebrating Remembrance Day. It appears that the younger population of the planet have little concern for events from several centuries ago.`
+	planet "Rand"
+		add spaceport `	The Remembrance Day preparations on Rand are unusually sparse compared to its population. It appears that most people have opted to return to their families on another planet to celebrate the day, though in one corner of the spaceport, you can hear a group of friends singing Alpha War shanties.`
+	planet "Sinter"
+		add spaceport `	The only sign that Remembrance Day is celebrated on this planet is a single monitor in the spaceport displaying the text, "Happy Remembrance Day!"`
+	planet "Valhalla"
+		add spaceport `	Despite the Deep's importance to defeating the Alphas, Remembrance Day is seldom celebrated in the region. The only thing you see out of the ordinary is a milk dish on the counter of an information booth.`
+
+
+event "remembrance day ends"
```
Comment (EjoThims) [inline data/human/events.txt:107]: All of these are just the current defaults again, yes?

**Review by EjoThims — APPROVED** [review]
Couple little things and questions, but this kind of thing is perfect for inclusion, imo. Nice little adjustments to make the world around you feel more alive :D

Amazinite makes a great point about the save bloat that I hadn't considered. I still really like the idea, but that definitely needs to be sorted.

**Review by Amazinite — CHANGES_REQUESTED** [review]
This seems like an inappropriate use of events to me. Given that these events will trigger every year, that means that for each year that a player plays their save file will get bigger and bigger because of these events, as events are added to the `changes` section of the save file in sequential order of when they were triggered. And given that the changes only last for two weeks, the player might not even notice the changed descriptions, so we'd basically be bloating their save file for nothing.

If the intention is to add extra flavor to the spaceport around Remembrance Day then I'd say that's exactly what spaceport news is for. It should be very easy to create a bunch of spaceport news that only offers during the lead up to Remembrance Day, and making use of spaceport news wouldn't pollute the save file with changes every year.

**Comment by McloughlinGuy** [comment]
I agree with amazinite, mainly because my computer seems to find ways to run out of storage space no matter what I do.

**Comment by MasterOfGrey** [comment]
I'm also in favour of using news instead of save file bloat, but while we're here...

@Amazinite since we have conditional text either PR'd or merged now, how difficult do you think it would be to add conditional planet descriptions?

**Comment by Amazinite** [comment]
It'd take a bit of work but wouldn't be too far out there.
Closing this since it won't be merged. If you go the news route then you can open a new PR. If we get planet descriptions that can change based on the player's conditions then we can revisit these exact descriptions without the need of events.

**Pull request closed without merging**